src/object/schedcontrol.c

```diff
@@ -71,7 +71,6 @@ static exception_t invokeSchedControl_Configure(sched_context_t *target, word_t
 #endif /* ENABLE_SMP_SUPPORT */
 
     if (target->scTcb && target->scRefillMax > 0) {
-        schedContext_resume(target);
```

Comment:
can you add an assert here to make sure it's true and document the assumption?

Reply:
Added this